cibuildwheel/pyodide.py

```diff
@@ -65,11 +67,63 @@ def install_emscripten(tmp: Path, version: str) -> Path:
     return emcc_path
 
 
-def install_xbuildenv(env: dict[str, str], pyodide_version: str) -> str:
+def search_xbuildenv(env: dict[str, str]) -> list[str]:
```

Comment:
Maybe `get_xbuildenv_versions`? It's not immediately obvious from this name how to interpret the return value.

Reply:
Makes sense to me, thanks – addressed in 8a8177c646f365bb8654f9f5560446acadf246bd.